### PR TITLE
Cleanup and reorganize examples

### PR DESCRIPTION
**Pull request by kadel**
As discussed in #136 this splits examples directory into two parts.

### docs/examples
Usage (spec) examples that are just showcasing Kedge features.
Those examples can sometimes use unnecessarily complicated constructs to highlight some of the Kedge features.

###  /examples
This should contain only real-world examples. It will become a library of Kedge Apps, where users can learn best practices from real examples. Those examples should use just constructs that make sense for given examples and keep the definition as clean as possible.

### CODE REVIEW

**Comment by kadel** [comment]
- updated  WordPress example to share secrets and configmap between mariadb and wordpress.
- added new example guestbook-demo. this is 3 tier web application (frontend, backend, db) with sources in https://github.com/kadel/guestbook-demo-backend and https://github.com/kadel/guestbook-demo-frontend more info can be found in https://github.com/kadel/guestbook-demo. This is ideal for demoing as you can do simple modification to source code and show updates and redeploys

File: examples/wordpress/wordpres.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: web
```
Comment (containscafeine) [inline examples/wordpress/wordpres.yaml]: nit: typo in file name - s/wordpres.yaml/wordpress.yaml

**Comment by containscafeine** [comment]
Looks fine to me, will let @pradeepto do an LGTM wrt UX and structure.

File: examples/wordpress/wordpres.yaml (hunk 1 above)
Comment (kadel) [inline examples/wordpress/wordpres.yaml]: thx, fixed

**Comment by pradeepto** [comment]
@kadel I have reviewed it. But would really like to have a native English speaker review this before we merge this since it is so important, especially the demo example.

@cdrage Can you have a look? Pending that, we should merge this.

File: docs/examples/basic/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+```yaml
+services:
+- name: wordpress
+  type: NodePort
+  ports:
+  - port: 8080
+    targetPort: 80
+```
+
+It is list of [service spec](https://kubernetes.io/docs/api-reference/v1.6/#servicespec-v1-core), which means that each app can have multiple services defined. Also see that ports are defined in the `services` field and not in the containers. You can choose to declare ports in the `containers.ports` as well but it is not required.
```
Comment (cdrage) [inline docs/examples/basic/README.md]: `it is a list of`

`can contain multiple services defined`

`Ports are defined in the services filed but not in the container field`

`declare ports in`

`as well, but it is not required`

File: examples/guestbook-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+# Guestbook demo
+
+This is demo application to show 3 tier web application.
```
Comment (cdrage) [inline examples/guestbook-demo/README.md]: `this is a demo application to show a 3-tier web application`

File: examples/guestbook-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+
+You can find more information https://github.com/kadel/guestbook-demo
```
Comment (cdrage) [inline examples/guestbook-demo/README.md]: `You can find more information at`

File: examples/wordpress/README.md [hunk 5]
```diff
@@ -1,19 +1,7 @@
...
-  type: NodePort
-  ports:
-  - port: 8080
-    targetPort: 80
-```
-
-It is list of [service spec](https://kubernetes.io/docs/api-reference/v1.6/#servicespec-v1-core), which means that each app can have multiple services defined. Also see that ports are defined in the `services` field and not in the containers. You can choose to declare ports in the `containers.ports` as well but it is not required.
 
+`mariadb.yaml` defines two Secrets (`database-root-password` and `database-user-password`) and one ConfigMap (`mariadb`).
+`wordprss.yaml` gets information on how to connect to database from `database-user-password` Secret and `mariadb` ConfigMap.
```
Comment (cdrage) [inline examples/wordpress/README.md]: `wordpress`

**Comment by kadel** [comment]
thx @cdrage  for looking at this

**Comment by cdrage** [comment]
LGTM. Merge whenever!

**Pull request merged**